tez-runtime-library/src/test/java/org/apache/tez/runtime/library/common/shuffle/impl/TestSimpleFetchedInputAllocator.java

```diff
@@ -66,7 +66,7 @@ public void testInMemAllocation() throws IOException {
     
     
     FetchedInput fi2 = inputManager.allocate(requestSize, compressedSize, new InputAttemptIdentifier(2, 1));
-    assertEquals(FetchedInput.Type.MEMORY, fi2.getType());
+    assertEquals(FetchedInput.Type.DISK, fi2.getType());
```

Comment:
hm, how is this related to the JDK17 upgrade?

Reply:
This test failed due to when i increased memory of surefire "-Xmx4096m -XX:+HeapDumpOnOutOfMemoryError". Anyway this value i will revert and check once. This value i modified as compile and run with jdk17 it is taking more time for some of the tests and some tests are timeout. I will revert this value to "xmx1024m" and check once. if any no issue then i revert this value and modify the test case accordingly.

Reply:
I cannot see this anymore, I guess it's working with 1024, resolving this